sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/EntityCategory.java

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.textanalytics.models;
+
+import com.azure.core.annotation.Immutable;
+import com.azure.core.util.ExpandableStringEnum;
+import com.fasterxml.jackson.annotation.JsonCreator;
+
+/**
+ * Gets the entity category inferred by the text analytics service's named entity recognition model.
+ * The list of available categories is described at
+ * See <a href="https://docs.microsoft.com/en-us/azure/cognitive-services/Text-Analytics/named-entity-types"></a>.
+ */
+@Immutable
+public final class EntityCategory extends ExpandableStringEnum<EntityCategory> {
```

Comment:
Does this have all the values mentioned here - https://docs.microsoft.com/en-us/azure/cognitive-services/Text-Analytics/named-entity-types